_fellows/james-villarrubia.md

```diff
@@ -3,13 +3,13 @@ id: james-villarrubia
 title: James Villarrubia
 image: james-villarrubia-headshot.png
 fellow_year: 2022
-project: []
+project: ['nasa-madi']
 agency: ["NASA"]
 hometown: Arlington, VA
 region: east
-bio_short: Creative technologist, supportive leader, and champion of positive social change through engineering.
-tags: ['east','2022','Data_Science_And_Analytics']
-skills: ['Data Science and Analytics']
+bio_short: Creative technologist and champion of positive social change through engineering
+tags: ['east','2022','Data_Science_And_Analytics','Software_And_Cloud_Infrastructure']
+skills: ['Data Science and Analytics','Software_And_Cloud_Infrastructure']
 ---
 
-James (he/him) believes his role as an engineer is to improve the world around him.  His 15-year career has been dedicated to solving critical community problems in the private and public sectors. James most recently served as the VP of Software Engineering for CipherHealth, enabling better patient outcomes through healthcare automation during the COVID-19 pandemic.  Before that, he served as the CTO of Werk, using A.I. to help Fortune 1000 companies improve productivity and equity through flexible work policies. James has also led startups focused in cybersecurity, cryptography, and educational technology.  His educational A.I. research project was a top-ranked competitor for the global IBM A.I. XPrize.  Before his startup journey, James spent six years in DC serving in policy and portfolio management roles at the White House, DOJ, and DOD. James earned a Master's of Public Policy and BS in Systems Engineering from the University of Virginia.  He is a certified Lean Six Sigma Black Belt. When he is able, James volunteers his time with alumni boards, voter outreach, and nonprofit theaters in his community. 
\ No newline at end of file
+James (he/him) believes his role as an engineer is to improve the world around him.  His 15-year career has been dedicated to solving critical community problems in the private and public sectors. James most recently served as the VP of Software Engineering for CipherHealth, enabling better patient outcomes through healthcare automation during the COVID-19 pandemic.  Before that, he served as the CTO of Werk, using A.I. to help Fortune 1000 companies improve productivity and equity through flexible work policies. James has also led startups focused in cybersecurity, cryptography, and educational technology. He holds three patents in cryptography.  His educational A.I. research project was a top-ranked competitor for the global IBM A.I. XPrize.  Before his startup journey, James spent six years in DC serving in policy and portfolio management roles at the White House, DOJ, and DOD. James earned a Master's of Public Policy and BS in Systems Engineering from the University of Virginia.  He is a certified Lean Six Sigma Black Belt. When he is able, James volunteers his time with alumni boards, voter outreach, and nonprofit theaters in his community.
```

Comment:
Appears to be a double space at the beginning of sentences. Please use single space.
Change "A.I." to "AI" throughout